src/Time.mo

```diff
@@ -2,12 +2,13 @@
 
 import Prim "mo:prim";
 module {
+    type Time = Nat64;
```

Comment:
As mentioned in a previous comment, I think we'll need to decide whether this module is supposed to be a Motoko abstraction (eventually) or a system abstraction. Maybe we need both, in which case we could call this type SystemTime? @nomeata, WDYT?

Reply:
I would start with the Motoko-idiomatic
```
type Time = Nat
```
and introduce a more low-level format only when and if needed.

Reply:
> Using Nat or Int would give users a wrong impression that things will be fine after 2554.

But it’s totally fine to express and talk about dates and times before 1970 and after 2554. The fact that `Time.now()` only reports times that are likely within the lifetime of the IC (more precise: this iteration of the System API) does not diminish that.

Reply:
@nomeata, SGTM.